tests/unittest_brain_pathlib.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+import astroid
+from astroid import bases
+from astroid.const import PY310_PLUS
+
+if not PY310_PLUS:
+    pytest.skip(
```

Comment:
Can you add tests for <3.10 as well? So just mark the tests for 3.10+ as such and add some tests for the old behaviour.

Reply:
Done!